docs/eog/eog-in-practice/cw23/feasibility-stage/fs-audience.rst

```diff
@@ -0,0 +1,55 @@
+.. _cw23-fs-audience:
+
+CW23 Audience
+=============================
+
+The Collaborations Workshop series brings together researchers, developers, innovators, managers, funders, publishers, leaders and educators to explore best practices and the future of research software.
+As such, our main target audience includes changemakers, advocates and ambassadors of all career stages and identities for each of these stakeholder groups who will continue and build off of the conversation and collaborations beyond the workshop.
+Our target audience comprises of people who can dedicate time to a three-day event as this will allow us to meet our :doc:`fs-goals-and-objectives`.
+By being multi-disciplinary, we can engage a wide range of people who can bring knowledge and skills back to their domains.
+
+The geographic location of the CW target audience is the UK/Europe.
+As CW23 will be hybrid, the target audience's geographic location will not be limited except by time zone - participants in Europe, Africa and eastern time zones of North and South America are more likely to participate remotely in a UK-working day event.
+This will allow us to continue to provide the opportunity to invite changemakers who would not normally be able to travel to the UK, and make the event more accessible.
+We also aim to engage people in other timezones in the discussions through social media and sharing of outputs.
+
+The typical number of participants at CWs in the past have been 80-90 for in-person CWs and 90-110 for online CWs.
+The number of participants will need to be capped to ensure equitable participation for both types of attendee, and an initial estimate would be to facilitate 70-80 in-person and 40-50 remote participants (or vice versa).
```

Comment:
This was what we planned but it turned out to be much more - I guess we will talk about what happened later in the in practice section.